internal/kafka/command_clientconfig_create.go

```diff
@@ -208,16 +208,8 @@ func (c *clientConfigCommand) setKafkaCluster(cmd *cobra.Command, configFile str
 }
 
 func (c *clientConfigCommand) setSchemaRegistryCluster(cmd *cobra.Command, configFile string) (string, error) {
```

Comment:
this function just calls c.getSchemaRegistryCluster(cmd) and there's no configFile involved. Can we just remove this function and replace it with c.getSchemaRegistryCluster(cmd)?

Reply:
Look further down?

Reply:
oh i think github cut the remaining - -